data.json

```
@@ -473,4 +473,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/61356005/104895611-1c547780-599c-11eb-8538-04f74005a4b4.png",
     "issueNumber": 250
   }
```

I think we should have a comma here. 
```suggestion
  },
```